app/src/main/java/org/systers/mentorship/view/fragments/ProfileFragment.kt

```diff
@@ -63,8 +63,10 @@ class ProfileFragment : BaseFragment() {
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         return when (item.itemId) {
             R.id.menu_edit_profile -> {
-                EditProfileFragment.newInstance(profileViewModel.user).show(fragmentManager,
-                        getString(R.string.fragment_title_edit_profile))
+                fragmentManager?.let {
+                    EditProfileFragment.newInstance(profileViewModel.user).show(
+                        it, getString(R.string.fragment_title_edit_profile))
+                }
```

Comment:
I think this was applied by a suggestion by Android Studio. You shouldn't apply those changes unless your PR is about fixing warnings.

Reply:
That's right, I'll revert the changes here.